api/scpca_portal/s3.py

```diff
@@ -14,18 +15,40 @@
 logger = get_and_configure_logger(__name__)
 aws_s3 = boto3.client("s3", config=Config(signature_version="s3v4"))
 
+TEST_INPUT_BUCKET_NAME = "scpca-portal-public-test-inputs/2024-04-19/"
+INPUT_BUCKET_NAME = TEST_INPUT_BUCKET_NAME if settings.TEST else "scpca-portal-inputs"
+
+
+def set_input_bucket(func: Callable) -> Callable:
```

Comment:
This is a possible strategy for handling the ability for the `load_data` command to pass an arbitrary `input_bucket_name` at run time that would be used by all functions in the `s3` module during the command's execution lifecycle.

Reply:
- bucket name should be in function signatures
- move input bucket default name to settings
- add Project::s3_input_bucket attribute, and populate it based on whether or not `input_bucket_name` was passed
- ComputedFile::s3_bucket name should stay